### PR TITLE
add action for misconfigured custom domain

### PR DESCRIPTION
**Pull request by T0MASD**
As part of PD#264983 found that the customer has misconfigured their custom domain, this message notifies the customer with the link to docs for custom domain setup

### CODE REVIEW

File: osd/custom_domain_misconfiguration.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Action required: update custom domain configuration",
+    "description": "Please verify the custom application domain '${CUSTOM_DOMAIN}' configuration to restore the cluster's operation. See documentation: https://access.redhat.com/articles/5599621",
```
Comment (cattias) [inline osd/custom_domain_misconfiguration.md:6]: Could we be a bit more specific on what the impact is there ?
Something like that maybe ?
The custom application domain '${CUSTOM_DOMAIN}' is misconfigured, generating alerts to Red Hat SRE. Please review it to restore the cluster's operation. See documentation : https://access.redhat.com/articles/5599621

Comment (T0MASD) [reply osd/custom_domain_misconfiguration.md]: Thanks for the review, sounds good, updated as you have suggested.

**Pull request merged**